modules/publication/php/files.class.inc

```diff
@@ -49,4 +49,20 @@ class Files extends \LORIS\Http\FilesPassthroughEndpoint
     {
         return "/files/";
     }
+
+    /**
+     * Send a notification for the download.
+     * 
+     * @param string $file The filename being downloaded
+     *
+     * @return void
+     * 
+     * @phan-suppress-next-line PhanUnusedProtectedMethodParameter
```

Comment:
You can wait until the tests run to see if it ignores comments but I'm pretty sure "suppress-next-line" needs to go immediately above the line.

Reply:
I believe it looks after the comment: https://github.com/phan/phan/wiki/Annotating-Your-Source-Code#phan-suppress-next-line